codeit/codeitHelp.py

```diff
@@ -17,8 +17,9 @@ def help():
     print(colored.red('Args list\n'))
     print(colored.yellow("-h -------------------------- " + colored.blue("To display help menu")))
     print(colored.yellow("-i <contest> ---------------- " + colored.blue("To initialize a contest directory")))
+    print(colored.yellow("-f <contest> ---------------- " + colored.blue("To create only files in the current directory level folder needed")))
```

Comment:
This is what I expected... Anyways, great job 👍

Reply:
Yups!